client/components/pagination/pagination-page.jsx

```diff
@@ -0,0 +1,99 @@
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+import Gridicon from 'gridicons';
+import { localize } from 'i18n-calypso';
+import React, { Component } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+
+class PaginationPage extends Component {
+	static propTypes = {
+		pageNumber: React.PropTypes.node.isRequired,
+		currentPage: React.PropTypes.number.isRequired,
+		totalPages: React.PropTypes.number.isRequired,
+		pageClick: React.PropTypes.func.isRequired,
+	}
+
+	clickHandler = ( event ) => {
+		event.stopPropagation();
+		switch ( this.props.pageNumber ) {
+			case '<--':
+				if ( this.props.currentPage - 1 < 1 ) {
+					return;
+				}
+				this.props.pageClick( this.props.currentPage - 1 );
+				break;
+			case '-->':
+				if ( this.props.currentPage + 1 > this.props.totalPages ) {
+					return;
+				}
+				this.props.pageClick( this.props.currentPage + 1 );
+				break;
+			default:
+				if ( this.props.currentPage === this.props.pageNumber ) {
+					return;
+				}
+				this.props.pageClick( this.props.pageNumber );
+				break;
+		}
+	}
+
+	render() {
+		const { numberFormat } = this.props;
```

Comment:
Here we already destructure `this.props`, and we could do the same as the previous function with `pageNumber`, `currentPage`, etc.